CHANGELOG.md

```diff
@@ -3,6 +3,35 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[1.2.0](https://github.com/sanger-tol/readmapping/releases/tag/1.2.0)] – Norwegian Ridgeback - [2023-12-20]
+
+### Enhancements & fixes
+
+- Restored recording read-groups (`@RG`) in the BAM/CRAM files.
+- Updated the CI procedure to use "sanger-tol" rather than "nf-core" names.
+- [crumble](https://github.com/jkbonfield/crumble) now used to compress the
+  PacBio HiFi alignments.
+- Execution statistics now under `pipeline_info/readmapping/` (to be consistent
+  with the other sanger-tol pipelines).
+- All resource requirements (memory, time, CPUs) now fit the actual usage. This
+  is achieved by automatically adjusting to the size of the input whenever
+  possible.
+
+### Parameters
```

Comment:
What about `use_work_dir_as_temp`?

Reply:
🤦🏼‍♂️ Completely missed that one... Thank you !